src/monthlychallenges/nov-2021/index.11tydata.js

```diff
@@ -77,7 +77,13 @@ module.exports = async function () {
 		}, {}),
 	).sort((a, b) => a.name.localeCompare(b.name));
 
+	const goals = [50000, 100000, 125000, 150000, 175000, 200000];
+	const completedGoals = goals.filter((g) => g <= totalCount);
+	const currentGoal = goals.find((g) => g > totalCount);
```

Comment:
will find always go through an array in ascending index order?

Reply:
yep it just finds the first one that matches!

Reply:
@tkshill  https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find